auth-plugin.c

```diff
@@ -52,7 +52,12 @@ int mosquitto_auth_plugin_init(void **user_data, struct mosquitto_auth_opt *auth
   GoSlice keysSlice = {keys, auth_opt_count, auth_opt_count};
   GoSlice valuesSlice = {values, auth_opt_count, auth_opt_count};
 
-  AuthPluginInit(keysSlice, valuesSlice, opts_count);
+  char versionArray[10];
+  sprintf(versionArray, "%i.%i.%i", LIBMOSQUITTO_MAJOR, LIBMOSQUITTO_MINOR, LIBMOSQUITTO_REVISION);
+
+  GoString version = {versionArray, strlen(versionArray)};
+
+  AuthPluginInit(keysSlice, valuesSlice, opts_count, version);
```

Comment:
This looks great! This is where I was getting stuck on my earlier attempts. I wasn't sure the cleanest way to pass the version information from C to Go